Tasks/Gradle/CodeAnalysis/SonarQube/server.ts

```diff
@@ -26,24 +26,29 @@ export class SonarQubeServer implements ISonarQubeServer {
     }
 
     public invokeApiCall(path:string):Q.Promise<Object> {
-        var defer = Q.defer<Object>();
 
+        tl.debug(`[SQ] Calling a REST API: ${path}`);
+
+        var defer = Q.defer<Object>();
         var options:any = this.createSonarQubeHttpRequestOptions(path);
 
         // Dynamic switching - we cannot use https.request() for http:// calls or vice versa
         var protocolToUse;
         switch (options.protocol) {
-            case 'http':
+            case 'http:':
```

Comment:
This is the actual fix - when parsing the url the protocol string is "https:" and not "https". A better fix for this would be to use a npm package like _request_ which takes care of these details. However, this isn't an opportune fix as this piece of code was manually tested quite a bit already for failures  - not the HTTPS path of course... :(